src/erlcloud_s3.erl

```diff
@@ -1141,6 +1141,8 @@ aws_region_from_host(Host) ->
     case string:tokens(Host, ".") of
         ["s3", _, _] ->
```

Comment:
I've fixed this in the previous PR which is not merged yet.
Will add it here also.